### PR TITLE
added feature flag for facility selection v2.2

### PR DESCRIPTION
**Pull request by grossb1**
## Description of change

Added feature flag for facility selection v2.2

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/18940

## Things to know about this PR

Added a feature flag to toggle the up-coming feature updates to facility selection, v2.2

### CODE REVIEW

File: config/features.yml [hunk 1]
```diff
@@ -388,4 +393,4 @@ features:
     actor_type: user
     enable_in_development: true
     description: >
-       Include state abbreviation when searching on comparison tool 
\ No newline at end of file
+      Include state abbreviation when searching on comparison tool
```
Comment (LindseySaari) [inline config/features.yml:396]: Was the change meant to be included?

Comment (grossb1) [reply config/features.yml:396]: My IDE must have automatically made that adjustment.  I will revert.

**Review by LindseySaari — APPROVED** [review]

File: config/features.yml (hunk 1 above)
Comment (grossb1) [reply config/features.yml]: Fixed!

Comment (LindseySaari) [reply config/features.yml]: Awesome, thanks!

**Pull request merged**